src/script/view_model/content/EmojiInputViewModel.ts

```diff
@@ -42,7 +62,7 @@ export class EmojiInputViewModel {
   // DO NOT USE COLON WITH LOWERCASE LETTERS IN THE SHORTCUTS, or you will prevent searching emojis.
   // For example, while :D should be replaced with unicode symbol, :d should allow searching for :dancer:
   /* eslint-disable sort-keys-fix/sort-keys-fix, no-multi-spaces */
-  static get INLINE_REPLACEMENT() {
+  static get INLINE_REPLACEMENT(): {name: string; shortcut: string}[] {
```

Comment:
```suggestion
  static get INLINE_REPLACEMENT(): Array<{name: string; shortcut: string}> {
```

Reply:
this one gives a lint error to use [] instead of Array<>

Reply:
![image](https://user-images.githubusercontent.com/63250054/86459042-744aca00-bd3b-11ea-8dba-c36f1538d1c8.png)

Reply:
ok :)